src/integrations/LiveChat/browser.js

```diff
@@ -29,40 +30,8 @@ class LiveChat {
     logger.debug('===in init Livechat===');
     window.__lc = window.__lc || {};
     window.__lc.license = this.licenseId;
```

Comment:
Isn't this also part of the loader logic?

Reply:
Updated